frontend/components/modals/LLCModal.tsx

```diff
@@ -0,0 +1,41 @@
+import { BaseModal } from './BaseModal'
+
+type LLCModalProps = {
```

Comment:
`LlcModalProps` if you're using acronyms in Pascal case you should only capital the first letter unless the acronym is 2 letters (like IOStream).
Here are the rules : https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions